### ISSUE TITLE
The COLLADA file created in Maya can not be correctly converted to glTF 2.0

### ISSUE THREAD

**Issue by cx20** [KhronosGroup/COLLADA2GLTF#57]
This Issue has been separated from https://github.com/KhronosGroup/glTF-Sample-Models/issues/83#issuecomment-310905402

I tried the simple skinning animation model below.
[EmaSimpleSkin.dae](https://github.com/cx20/jsdo-static-contents/blob/master/models/collada/EmaSimpleSkin/EmaSimpleSkin.dae) : This is a COLLADA file exported from Maya using OpenCOLLADA 2016. This sample was donated by @emadurandal
I converted it into glTF 2.0 format using [COLLADA2GLTF](https://github.com/KhronosGroup/COLLADA2GLTF).

The following is the display result by Three.js. As animation movement the glTF 1.0 model is more correct.
Because [WalkingLady.dae](https://github.com/KhronosGroup/glTF-Sample-Models/blob/master/sourceModels/WalkingLady/WalkingLady.dae) has also been exported from Maya, I think that it includes the same problem.

[Three.js + glTF 1.0 model result:](http://jsdo.it/cx20/coRj)
![emasimpleskin_threejs_gltf1](https://user-images.githubusercontent.com/1833979/27516485-f37ef488-59f5-11e7-8467-48a024c393c7.gif)

[Three.js + glTF 2.0 model result:](http://jsdo.it/cx20/q0cx)
![emasimpleskin_threejs_gltf2](https://user-images.githubusercontent.com/1833979/27516487-fb2fb85c-59f5-11e7-85da-1fde932e3605.gif)

I think that models including skinning animation have not been converted correctly.

**Comment by lasalvavida** [comment]
Generated with #81. Also worth noting that the Cesium Milk Truck (the only other sample model with an ANGLE transform) correctly identifies that it doesn't need to be flipped, and continues rolling forward instead of backwards.

![ezgif-4-2e6969941d](https://user-images.githubusercontent.com/2489570/29751361-6280ea4e-8b1a-11e7-83b4-d6f96ead16df.gif)
[EmaSimpleSkin.zip](https://github.com/KhronosGroup/COLLADA2GLTF/files/1254475/EmaSimpleSkin.zip)

**Comment by lasalvavida** [comment]
I also pushed out a new release with this fix, so the compiled releases should include this shortly.

**Comment by cx20** [comment]
I confirmed that EmaSimpleSkin.gltf is correctly displayed in Three.js.
http://jsdo.it/cx20/YRG5

If WalkingLady.gltf can be displayed in the same way, this Issue will be closed :)

**Comment by cx20** [comment]
I tried to convert from WalkingLady.dae to WalkingLady.gltf.
```
COLLADA2GLTF-bin -i WalkingLady.dae WalkingLady.gltf -v 2.0
```
[WalkingLady.zip](https://github.com/KhronosGroup/COLLADA2GLTF/files/1264837/WalkingLady.zip)

However, the problem of skinning does not seem to be solved.
![image](https://user-images.githubusercontent.com/1833979/29891428-fa6ffa6e-8e05-11e7-9833-003a00e3d13b.png)

**Comment by lasalvavida** [comment]
@cx20, it isn't really the same problem. The issue with the other model was to do with the direction of the rotation around an axis during the animation. Something happens with the skinning of WalkingLady, but it is inconsistent across renderers, so I'm not sure if it's a renderer problem or a conversion problem. Or maybe it is a renderer problem but there is something we can do in the converter to make the file more agreeable. I haven't looked into it very deeply.

**Comment by cx20** [comment]
@lasalvavida Previously glTF-Blender-Exporter has been fixed with similar problems. It might be helpful there.
https://github.com/KhronosGroup/glTF-Blender-Exporter/issues/22

**Comment by cx20** [comment]
I think the WalkingLady.gltf problem will be fixed in the below PR.
https://github.com/KhronosGroup/COLLADA2GLTF/pull/96

**Comment by lasalvavida** [comment]
Closed by #96. Thanks for the reminder @cx20!
